packages/dredd-transactions/scripts/prepack.js

```diff
@@ -9,8 +9,13 @@ const rimraf = require('rimraf');
 
 const PACKAGE_DIR = path.resolve(__dirname, '..');
 const SYMLINKS_LOG = path.join(PACKAGE_DIR, 'prepack-symlinks.log');
-const APIB_PARSER_PATH = path.join(PACKAGE_DIR, 'node_modules', '@apielements', 'apib-parser');
 
+// Resolve "@apielements/apib-parser" module location against both
+// root-level and relative "node_modules" directory.
+const APIB_PARSER_PATH = [
+  path.join(PACKAGE_DIR, 'node_modules', '@apielements', 'apib-parser'),
+  path.join(PACKAGE_DIR, '../..', 'node_modules', '@apielements', 'apib-parser')
```

Comment:
With this addition it symlinks the `@apielements/apib-parser` without exceptions on my machine. I'm preserving the previous logic for backward-compatibility, yet it always installs `@apielements/apib-parser` on the root-level of the monorepo in all these scenarios:

- using `lerna bootstrap`
- explicitly installing `lerna add @apielements/apib-parser --scope=dredd-transactions`
- removing `yarn.lock` and trying both of the above.